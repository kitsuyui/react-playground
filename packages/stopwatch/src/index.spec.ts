import { describe, expect, it } from 'vitest'

import * as module from '.'

describe('module', () => {
  it('exports', () => {
    expect(module).toHaveProperty('MinimalStopwatch')
    expect(module).toHaveProperty('StopwatchContext')
    expect(module).toHaveProperty('StopwatchContextProvider')
  })
})
