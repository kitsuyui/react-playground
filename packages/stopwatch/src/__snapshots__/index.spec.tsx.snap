// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`render MinimalStopwatch 1`] = `
<DocumentFragment>
  <div
    style="width: 6em; font-family: monospace;"
  >
    <form>
      <span>
        00:00.000
      </span>
      <br />
      <button
        name="reset"
        type="submit"
      >
        Reset
      </button>
      <button
        type="submit"
      >
        Start
      </button>
    </form>
  </div>
</DocumentFragment>
`;
